docs/ingestion/compaction.md

```diff
@@ -217,7 +217,7 @@ Druid supports two supported `inputSpec` formats:
 |-----|-----------|--------|
 |`segmentGranularity`|Time chunking period for the segment granularity. Defaults to 'null', which preserves the original segment granularity. Accepts all [Query granularity](../querying/granularities.md) values.|No|
 |`queryGranularity`|The resolution of timestamp storage within each segment. Defaults to 'null', which preserves the original query granularity. Accepts all [Query granularity](../querying/granularities.md) values.|No|
-|`rollup`|Whether to enable ingestion-time rollup or not. Defaults to 'null', which preserves the original setting. Note that once data is rollup, individual records can no longer be recovered. |No|
+|`rollup`|Set the value to `true` to enable compaction-time rollup. Once the data is rolled up, you can no longer recover individual records. Defaults to 'null', which preserves the original setting.|No|
```

Comment:
This one requires more explanation! If the input segments are already aggregated, then it make sense that the outputs also be aggregated. If the input segments are aggregated, it is hard to see why we'd *not* want to continue to aggregate, or what happens to the input partial aggregations. Do I have to provide reduce functions to convert the aggregates to simple values?

Then, if the input is detail, and the output is rolled-up, I have to provide dimensions and measures (aggregates), else, I would imagine, compaction has nothing to work with.

I would suggest we a) strongly encourage the default (keep input setting), and b) create an entire section to explain what happens if the user transitions from rollup --> detail or detail --> rollup. Without that, the meaning of this setting is very unclear!

Reply:
Creating a new section to explain what happens when a user enables rollup is out of scope for this ticket. We can address it on a separate ticket.